contributing.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: page
+title: Contributing
+show_sidebar: false
+menubar: menu
+hide_hero: true
+# TODO: Publish this page when it is finalized, adding a link for it to the menubar
+published: false
```

Comment:
I set this page to not be published, so it won't be available. This page just contains content that used to be on the home page that I think would work well in a contributing guide.